frontend/package.json

```diff
@@ -20,6 +21,7 @@
     "react": "^17.0.1",
     "react-card-flip": "^1.0.11",
     "react-dom": "^17.0.1",
+    "react-hook-form": "^6.15.5",
```

Comment:
Again why do we need this extra package

Reply:
Actally this  is a new feature  or package that React released lil time ago , and this is best suited for Functional Components
Due to addition of Hooks In React , this package is becoming more usable and popular  , so I decided to use this